frontend/javascripts/test/libs/format_utils.spec.ts

```diff
@@ -0,0 +1,51 @@
+import test from "ava";
+import { formatNumberToArea, formatNumberToLength, formatNumberToVolume } from "libs/format_utils";
+import { Unicode } from "oxalis/constants";
+
+test("Format number to length", (t) => {
+  const { ThinSpace } = Unicode;
+  const lengthsInNm = [107e-3, 107, 107e3, 107e6, 107e9, 107e12];
```

Comment:
please also use some other numbers to make the tests a bit more thorough. e.g., I'd like to see the following result strings:
`1${ThinSpace}mm`
`12${ThinSpace}nm`
`0.01${ThinSpace}pm`
`1234${ThinSpace}km`

Reply:
I tried :) please let me know if I can still improve the tests!